### DISCUSSION TITLE
Offset wizard always grayed out

### DISCUSSION THREAD

**Discussion by killer78887** [ppy/osu#21291] [Q&A] [answered]
I am not sure if its supposed to be like that, but the offset wizard is always greyed out and I cant use it.
Is it not implemented yet or a bug?
I am on android
![SmartSelect_20221117_182854](https://user-images.githubusercontent.com/44791392/202632967-a930e85a-e1e3-424a-a77f-72b0646f6e73.jpg)

**Comment by NotGumballer91** [comment]
Not implemented yet

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/9866
